editMember.php

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="Edit Member Form" />
+    <meta name="keywords" content="HTML, CSS, JavaScript" />
+    <meta name="author" content="MSP Group 20" />
+    
+    <link href="styles/editMember.css" rel="stylesheet" type="text/css" />
+
+    <title>Edit Member Form</title>
+
+    
+    
+  </head>
+  <body>
+
+
+<h2>Edit Member Form</h2>
+
+<form id="editMemberform"> 
+
+    
+        <!-- <legend style="font-size:19px"><strong>Member Details</strong></legend> -->
+   <p>
```

Comment:
You can wrap `<div>` and give them a style instead of `<p>` here. I showed Kratti how to layout everything top down in #10. You can have a look at that in there